.github/workflows/reusable-performance.yml

```diff
@@ -287,6 +291,9 @@ jobs:
           name: performance-artifacts${{ inputs.memcached && '-memcached' || '' }}-${{ github.run_id }}
           path: artifacts
           if-no-files-found: ignore
+        env:
+          CODEVITALS_PROJECT_TOKEN: ${{ secrets.CODEVITALS_PROJECT_TOKEN }}
+          HAS_CODEVITALS_PROJECT_TOKEN: ${{ !! secrets.CODEVITALS_PROJECT_TOKEN && secrets.CODEVITALS_PROJECT_TOKEN != '' }}
```

Comment:
Note to reviewers: This is temporary, just to debug whether the secret is set. But there is a chance it's not working (see https://github.com/WordPress/wordpress-develop/pull/7490#issuecomment-2392190473).